docs/api/securing-api.md

```diff
@@ -0,0 +1,88 @@
+
+![An image of a new key being generated on OneBlink Console](../pics/DeveloperKeyCreation.png)
+
+After generating this key, you should see something like this:
+
+![An example of what a developer key looks like after generation on the OneBlink Console](../pics/DeveloperKeyExample.png)
+
+I've blacked out my keys obviously but you should have 3 rows of data, 1 visible but the other 2 are hidden with dots. Now that we have the key, let us jump back into the code. 
+
+## Updating your Code
+
+Now we need to go into our code and create some checks to ensure that the user is valid. The steps we will go through to achieve this are:
+
+- Create a new FormsApp Object
+- Use the VerifyJWT function from it
+
+Now let us go down the list!
+
+### Create a new FormsApp Object
+
+Firstly, we need to use the FormsApp object from the OneBlink SDK so add this code to the top of your file you are working on:
+
+```js
+const formsApp = require("@oneblink/sdk/tenants/oneblink");
+```
+
+If you are using ES Modules, you can instead write:
+
+```js
+import { FormsApps } from '@oneblink/sdk/tenants/oneblink'
+```
+
+You can find out more about converting your functions to ES Module with [this article](./upgrading-to-es-modules.md).
+
+Now that we have imported formsApp from the OneBlink SDK, we will need to grab our key that we generated on the Console before. 
+Here we will write:
+
+```js
+  const formsAppsClient = new FormsApps({
+    accessKey: YOUR_ACCESS_KEY,
+    secretKey: YOUR_SECRET_KEY,
+  })
+```
+replacing the values with the respective values from the Console. You can place this inside, or outside of your function. After this, in your function at the very beginning, you can use this code:
+
+```js
+    const authenticationHeader = req.headers.authorization
+    if (typeof authenticationHeader !== 'string'){
+      return res.setStatusCode(400).setPayload({
+        message: "Error, no user logged in. User must be logged in."
+      });
+    }
+
+    const token = authHeader.split(' ')[1]
+    const jwtPayload = await formsAppClient.verifyJWT(token)
+```
+
+Alright so let's break this down.
+First, we are grabbing the authentication header from the request that was sent to the API from the request's header. After this, we are checking to see if the header is of type string. If it is not, the user has NOT logged in! That means we cannot authenticate this user so we return a 400 on the `res` object by `setPayload(400)` to return a 400 HTTP code and return `setPayload({message: 'Error, no user logged in. User must be logged in.'})` as the payload with an error message. If there is an authentication header, we can split it so we grab the JWT token. After this, then we can run the `verifyJWT()` function with the token to verify who the user is! 
```

Comment:
link to this as well? https://oneblink.github.io/sdk-node-js/classes/oneblink.FormsApps.html#verifyJWT